build/schemas/tax/region.json

```diff
@@ -344,7 +208,7 @@
           "description": "List of tax category codes that can be used when this scheme is\napplied."
         },
         "note": {
-          "$ref": "#/$defs/Note",
+          "$ref": "https://gobl.org/draft-0/org/note",
           "title": "Note",
           "description": "Notes defines messages that should be added to a document\nwhen this scheme is used."
```

Comment:
Just noticed that this description was in plural when, at the moment, one single note can be set here. This was pre-existing discrepancy but perhaps we can fix it now?

Reply:
Good find! Yes, I'll fix.